RadixWallet/Features/EntitySecurityProblems/EntitySecurityProblems.swift

```diff
@@ -8,9 +8,16 @@ public struct EntitySecurityProblems: Sendable, FeatureReducer {
 
 		let kind: Kind
 		var problems: [SecurityProblem] = []
+		fileprivate let loadProblems: Bool
 
-		public init(kind: Kind) {
+		public init(kind: Kind, problems: [SecurityProblem]? = nil) {
```

Comment:
Initially I wanted to inject the `[SecurityProblems]` every time, forcing the client of it to have them loaded. However, this is used by `PersonaFeature.View`, which is used in different contexts in which handling the load of the security problems will be troublesome. Therefore, decided to leave this component as one that could either receive the problems (as in Home rows) or load them by itself (Personas).

Reply:
Update: I believe it is worth going with the plain SwiftUI solution for this, and have the `problems` always injected. Will work on improvement tomorrow.